vips/vip-315/bscmainnet.ts

```diff
@@ -0,0 +1,45 @@
+import { vweETH, vwstETH } from "../../multisig/proposals/ethereum/vip-032";
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+const vip315 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-315 Risk Parameters Adjustments (weETH, wstETH)",
+    description: `This VIP will perform the following Risk Parameter actions as per Chaos Labs’ latest recommendations in this Venus community forum publication: [Chaos Labs - Risk Parameter Updates - 05/27/24](https://community.venus.io/t/chaos-labs-risk-parameter-updates-05-27-24/4364/1).
+
+- [weETH (Liquid Staked ETH pool)](https://etherscan.io/address/${vweETH})
+  - Increase supply cap, from 15K to 50K weETH
+  - Increase borrow cap, from 7.5K to 25K weETH
+- [wstETH (Liquid Staked ETH pool)](https://etherscan.io/address/${vwstETH})
+  - Increase borrow cap, from 4K to 10K wstETH
+
+Complete analysis and details of these recommendations are available in the above publication.
+
+VIP simulation: https://github.com/VenusProtocol/vips/pull/294
+
+#### Disclaimer for Ethereum VIPs
+
+Privilege commands on Ethereum will be executed by the [Guardian wallet](https://etherscan.io/address/0x285960C5B22fD66A736C7136967A3eB15e93CC67), until the [Multichain Governance contracts](https://docs-v4.venus.io/technical-reference/reference-technical-articles/multichain-governance) are deployed. If this VIP passes, this multisig transaction will be executed. Otherwise, it will be rejected.`,
```

Comment:
```suggestion
Privilege commands on Ethereum will be executed by the [Guardian wallet](https://etherscan.io/address/0x285960C5B22fD66A736C7136967A3eB15e93CC67), until the [Multichain Governance contracts](https://docs-v4.venus.io/technical-reference/reference-technical-articles/multichain-governance) are deployed. If this VIP passes, [this](https://app.safe.global/transactions/tx?safe=eth:0x285960C5B22fD66A736C7136967A3eB15e93CC67&id=multisig_0x285960C5B22fD66A736C7136967A3eB15e93CC67_0xcde4d485220c45190037c5897bf8ca2aee6d6e05d8a817497a161b297870b18b) multisig transaction will be executed. Otherwise, it will be rejected.`,
```